internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -89,9 +89,9 @@ type Opts struct {
 	SkipMongosh         bool
 	runMongoShell       bool
 	mongoShellInstalled bool
-	defaultValue        bool
+	DefaultValue        bool
 	Confirm             bool
-	store               store.AtlasClusterQuickStarter
+	Store               store.AtlasClusterQuickStarter
```

Comment:
[nit] you should not export the store, this is going to be troublesome later when set up also needs to do registration and login, which are different stores from quickstart